src/app/news/news.scss

```diff
@@ -1,14 +1,12 @@
-$max-width: 640px;
+@import '../variables';
 
 .view-container {
   display: grid;
-  grid-template-rows: auto 1fr;
   grid-template-areas: 'form' 'news';
-  justify-content: center;
-  grid-template-columns: minmax(auto, $max-width);
+  grid-template-columns: 1fr 2fr;
 
   .new-post-form {
-    grid-area: form;
+    grid-area: 'form';
```

Comment:
This is not valid CSS.  `grid-area` should not have quotes.

![image](https://user-images.githubusercontent.com/9203229/54893245-39e15480-4e8b-11e9-8535-b75562e3cc5f.png)